### ISSUE TITLE
ComboBox: NVDA screen reader does not consistently announce the selected item.

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#11109]
### Bug Description

Go to:
https://sap.github.io/ui5-webcomponents/components/ComboBox/

Select any item. Screen reader does not announce the selected item. 

![Image](https://github.com/user-attachments/assets/ecea4bfa-1280-4080-9b46-60da87da1f49)

### Affected Component

ComboBox

### Steps to Reproduce

1.
2.
3.
...

### Log Output, Stack Trace or Screenshots

https://github.com/user-attachments/assets/b791408b-ed58-4652-a5d7-f18bbc01566e

### Priority

None

### UI5 Web Components Version

2.x

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Dear. @gurkirpalgill, 

I saw your video and you are testing the control improperly. You are using the mouse to select values. You should only use keyboard - this way everything is announced properly. 

Kind reagrds,
Georgi UI5 Dispatcher
